OpenEdXMobile/src/main/java/org/edx/mobile/http/notifications/DialogErrorNotification.java

```diff
@@ -10,26 +10,27 @@
 
 import com.joanzapata.iconify.Icon;
 
+import org.edx.mobile.base.BaseFragment;
```

Comment:
There is an unused import in this file:
`import android.support.v4.app.FragmentManager;`.